### PR TITLE
move notes feature page, improve readme, and cleanup

### PR DESCRIPTION
**Pull request by vincanger**
- moved notes feature off landing page
- improved readme by adding more details and links to wasp docs
- put notes link in navbar 
- general code cleanup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ## Wasp Cursor IDE Template
 
-This is a template to help you build your Wasp app with the [Cursor IDE](https://cursor.sh/).
+This is a basic starter template for [Wasp](https://wasp-lang.dev/) apps but with a couple of modifications to give you the best experience when using the [Cursor IDE](https://cursor.sh/) for development. Specifically, a `.cursorrules` file optimized for Wasp apps, and Wasp example code curated specifically for Cursor.
 
 ## Getting Started
 
 1. Make sure you have Wasp installed: `curl -sSL https://get.wasp-lang.dev/installer.sh | sh -s`
-2. Run `wasp new -t cursor-template` to fetch the template (coming soon. until then, clone the repo).
+2. Clone this repo: `git clone https://github.com/wasp-lang/cursor-template.git`
```
Comment (Martinsos) [inline README.md]: What about Github's "Use as a template" button? I saw Roke has that: https://roke.dev/guide . Should we also consider that? Just asking, I dont' konw.

File: README.md [hunk 2]
```diff
@@ -19,6 +19,52 @@ The `.cursorrules` file is our attempt at fixing the common mistakes the AI assi
...
+
+The project is structured to offer just enough context for the AI to help you with Wasp-specific code, and to get you started with building your app quickly, but without overwhelming you with too much code.
+
+In the sections below, we'll go through each directory and explain why we've included them and what they do.
+
+```
+src/
+├── auth/                          // Auth-related files
+├── client/                        // Client-only components and the Main.tsx app wrapper
+├── exampleNotesFeature/           // Example feature code
```
Comment (Martinsos) [inline README.md]:
```suggestion
├── exampleNotesFeature/           // Example feature code (for Cursor)
```

File: README.md [hunk 3]
```diff
@@ -19,6 +19,52 @@ The `.cursorrules` file is our attempt at fixing the common mistakes the AI assi
...
+### Auth
+
+The `auth` directory contains the login and signup pages. They import and take advantage of Wasp's Auth API to handle the full-stack auth logic for you. E.g., after defining the Auth methods you'd like to use in `main.wasp`, Wasp will generate the login and signup forms for the defined methods, and handle the full-stack auth logic for you, including validation, error handling, etc.
+
+For more info on Wasp Auth, check out the [Wasp Auth docs](https://wasp-lang.dev/docs/auth/overview).
+
+### Client
+
+The `client` directory contains the client-only components and the `Main.tsx` app wrapper. It also contains the `Main.css` file, which imports Tailwind CSS so you can can get started styling your app with Tailwind without having to configure it yourself.
+
```
Comment (Martinsos) [inline README.md:58]: You went quite detailed here! I would probably just give them a single link to Wasp docs and tell them to learn more about stuff like auth, main.wasp and similar there. But if you think this is the right amount of info, why not, that is also ok.

**Review by Martinsos — APPROVED** [review]
LGTM @vincanger ! Left a tiny one or two comments, but all good from my side.

File: README.md (hunk 1 above)
Comment (vincanger) [reply README.md]: Yeah thats a good idea because then the get a clean commit history.

File: README.md (hunk 3 above)
Comment (vincanger) [reply README.md:58]: I assumed it would help people get started faster...

**Pull request merged**